docs/howto/filesystem-management/data-transfer.md

```diff
@@ -0,0 +1,138 @@
+(migrate-data)=
+# Transfer data across filestores
+
+This documentation covers how to transfer data between filestores.
```

Comment:
Let's note this works across cloud providers.

Reply:
Resolved by https://github.com/2i2c-org/infrastructure/pull/5284/commits/3161f51f5da584463119efa9176accce5604ee89